### ISSUE TITLE
I am not sure what the input of the loss function is.

### ISSUE THREAD

**Issue by gyc521zsc** [ppriyank/-Online-Soft-Mining-and-Class-Aware-Attention-Pytorch#2]
I have a question I would like to ask you, I use your loss function, the result is 0. This is the last two layers of my network.
![image](https://user-images.githubusercontent.com/50411057/68192162-e7a10e00-ffeb-11e9-99c2-10ef6cb3c4b2.png)
![image](https://user-images.githubusercontent.com/50411057/68192293-13bc8f00-ffec-11e9-88c2-99d78933542c.png)
